### ISSUE TITLE
archived expansion modoboa-pdfcredentials is outdated and causes erro 500: (internal server error)

### ISSUE THREAD

**Issue by mathl79** [modoboa/modoboa#3078]
# Impacted versions

* OS Type: Debian/Ubuntu
* OS Version: 22
* Database Type: PostgreSQL 
* Database version: X.y
* Modoboa: 2
* installer used: Yes
* Webserver: Nginx

# Steps to reproduce
add extansion pdfcredentials to settings.txt and install via installer script

# Current behavior
installation finishes, but server responds with Error 500(internal serer error)
django module is missing due to error at the installprocess of pdfcredentials

repo of pdfcredentials should be opend again and django dependency need to be upgraded.

**Comment by Spitfireap** [comment]
Pdfcredentials is part of the core modoboa package since 2.1.0. This why it has been archived.

**Comment by mathl79** [comment]
good to know, thanks!

I stumbled over this as I tried to upgrade  my modoboa installation leading to "internal Server error" and after that also failing to upgrade from an old backup because of automatic applied modoboa-pdfcredentials setting.

Maybe it is important to say that I did not put the PDF credentials extension into the settings by myself.
It was is set automatically, which prevents the recovery process in this scenario upgrading from an version prior to 2.1.0, when PDF credentials extension was activated in the old version.

maybe this can be catched within the modoboa-installer?

br


⁣BlueMail for Android herunterladen ​
